src/tests/functional/shared_test_classes/include/shared_test_classes/base/benchmark.hpp

```diff
@@ -0,0 +1,325 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cctype>
+#include <iostream>
+
+#include <nlohmann/json.hpp>
+#include "layer_test_utils.hpp"
+#include "ov_subgraph.hpp"
+
+namespace ov {
+namespace test {
+
+using json = nlohmann::json;
+
+class BenchmarkLayerTestReporter {
+public:
+    static constexpr const char* benchmarkReportFileName = "benchmark_layers.json";
+
+    explicit BenchmarkLayerTestReporter(bool is_readonly)
+        : is_readonly_{is_readonly} {
+        if (is_readonly) {
+            try {
+                report_file_.open(benchmarkReportFileName, std::ios::in);
```

Comment:
usually, we use xml format for the reporting (pugixml is linked directly to the shared test lib)

Reply:
Change done